p11-kit/client.c

```diff
@@ -163,7 +163,7 @@ p11_client_module_cleanup (void)
 	for (; state != NULL; state = next) {
 		next = state->next;
 		p11_rpc_transport_free (state->rpc);
-		p11_virtual_unwrap (state->wrapped);
+		p11_virtual_unwrap ((CK_FUNCTION_LIST *)state->wrapped);
```

Comment:
Not a fault of this PR, but we probably should change `p11_virtual_unwrap` to take `CK_FUNCTION_LIST_3_0 *` instead of `CK_FUNCTION_LIST *` to be consistent with `p11_virtual_wrap`.

Reply:
but `p11_virtual_wrap` returns `CK_FUNCTION_LIST *` should I change all of these?

Reply:
this change would be huge and would require to retype the entire module.c and probably another modules as well, like proxy.c ... I think if we want to do this, then in a separate PR

Reply:
Let's postpone it (or do it in a separate PR).